api-gateway/src/main/resources/static/scripts/bill-details/bill-details.controller.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+angular.module('billDetails')
+    .controller('BillDetailsController', ['$http', '$stateParams', function ($http, $stateParams) {
+        var self = this;
+
+        $http.get('api/gateway/bill/' + $stateParams.billId).then(function (resp) {
+            self.bills = resp.data;
+        });
+
+        $http.delete('api/gateway/bill/' + $stateParams.billId).then(function (resp) {
+            self.bills = resp.data;
+        });
+    }]);
```

Comment:
You should maybe consider using a more generalized title to better match what is being added. (Ex: Setup Billing Service Front-End)